hledger-web/Hledger/Web/Foundation.hs

```diff
@@ -100,9 +102,11 @@ type Form x = Html -> MForm (HandlerT App IO) (FormResult x, Widget)
 instance Yesod App where
   approot = ApprootMaster $ appRoot . settings
 
-  makeSessionBackend _ =
+  makeSessionBackend _ = do
+    hledgerdata <- getXdgDirectory XdgData "hledger"
```

Comment:
For this runtime-generated file, is XdgData (`~/.local/share`, `C:/Users/<user>/AppData/Roaming`) the best place, or is XdgCache (`~/.cache`, `C:/Users/<user>/AppData/Local`) ? (I see XDG_RUNTIME_DIR is not yet supported, a pity.)

Reply:
You're right. Updated.